src/auth/__tests__/authService.test.js

```diff
@@ -269,18 +283,87 @@ describe('authService', () => {
     it('should call sessionStorage.setItem with the right arguments', async () => {
       const setSpy = jest.spyOn(Storage.prototype, 'setItem');
       expect.assertions(2);
-      await authService.fetchApiToken(mockUser);
+      await authService.fetchAndStoreApiToken(mockUser);
 
       expect(setSpy).toHaveBeenCalledTimes(1);
       expect(setSpy.mock.calls[0]).toMatchInlineSnapshot(`
         Array [
           "apiToken",
-          "71ffd52c-5985-46d3-b445-490554f4012a",
+          "${apiToken}",
         ]
       `);
     });
   });
 
+  describe(`Api tokens are fetched again after user tokens are renewed.
+            After silent renew completes, the _userLoaded event is raised and`, () => {
```

Comment:
The test name seems to end abruptly. I'm not sure should it continue more or is the final "and" unnecessary.

Reply:
The test group title is supposed to work as a prefix for the actual tests. The output of the tests would be combination of those. For example

describe(`Function xyz() ")
   it("returns true when arguments have something")
   it("returns false when arguments have something else")
   
Would output   
  
'Function xyz() returns true when arguments have something'
'Function xyz() returns false when arguments have something else'

I changed the titles a bit. Added "..." to indicate that text continues below.

Reply:
Ahh, of course. I missed that.